### PR TITLE
Create german-council-of-economic-experts.csl

### PR DESCRIPTION
**Pull request by Irazall**
The German Council of Economic Experts provides an own citation style. In order to distribute this style to all contributors, I hereby want to upload this style.

### CODE REVIEW

**Comment by rmzelle** [comment]
@Irazall, is there an (online) description of this citation format we can link to?

**Comment by Irazall** [comment]
No, there is none.

**Comment by adam3smith** [comment]
any way you could make this available, even if just as an attachment here? We don't like having styles for which we have no way to check accuracy when issues arise.
Also, how many contributors are we talking here? This is by invitation only, isn't it?

**Comment by Irazall** [comment]
We have an intern documen in germant:
[Literatur.pdf](https://github.com/citation-style-language/styles/files/831406/Literatur.pdf)

There is no other contributor except me. But just in case it is by inviation only.

**Comment by Irazall** [comment]
Ah, ok. There are around 25 users intern. Furthermore we accept reports from externals that we will publish as working papers on our website. In numbers there are about 15 external reports each year.

**Comment by adam3smith** [comment]
And the working paper series (for lack of a better word) is open for unsolicited submissions? I.e. econ prof XYZ could just send in a report and have it considered for publication? 
That'd do for us. Sorry for the many questions, trying to establish if this falls under https://github.com/citation-style-language/styles/wiki/Criteria-for-Accepting-Styles

**Comment by Irazall** [comment]
Well, not really. It is more informel. Sometimes the council searches for contributors, sometimes the contributors ask if the council is in need of some professional expertise.

**Comment by adam3smith** [comment]
Hmm -- then it seems like it'd be very easy for you to just host the style locally and point contributors to it, no? You're communicating with them before they're writing anyway. @rmzelle , thoughts?

**Comment by Irazall** [comment]
Actually I don't know how to host it locally. I sure can upload it on our website, but how can I make zotero to update this style automatically on a regular basis?

**Comment by rmzelle** [comment]
> how can I make zotero to update this style automatically on a regular basis?

I think Zotero currently only auto-updates styles hosted in the repository, so if you host this yourself users would have to reinstall the style to get the latest version whenever you make a change.

**Comment by Irazall** [comment]
Since this is not what we want, I ask you sincerely to add our style to your repository

**Comment by rmzelle** [comment]
Sure. @Irazall, would it make more sense to change the style's title and file name to the German equivalent, since it's a German style?

**Comment by Irazall** [comment]
Thanks @rmzelle. 
We also publish in english. And since we also have international conferences "German Council of Economic  Experts" is a popular name. At least in our research circle ;)

File: german-council-of-economic-experts.csl [hunk 1]
```diff
@@ -0,0 +1,310 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="in-text" default-locale="de-DE" demote-non-dropping-particle="sort-only">
+  <!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>German Council of Economic Experts</title>
+    <title-short>GCEE</title-short>
+    <id>http://www.zotero.org/styles/german-council-of-economic-experts</id>
+    <link href="http://www.zotero.org/styles/german-council-of-economic-experts" rel="self"/>
```
Comment (rmzelle) [inline german-council-of-economic-experts.csl]: Can you add a "documentation" link here to https://www.sachverstaendigenrat-wirtschaft.de/? See https://github.com/citation-style-language/styles/wiki/Style-Requirements#10---documentation-link

File: german-council-of-economic-experts.csl [hunk 2]
```diff
@@ -0,0 +1,310 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="in-text" default-locale="de-DE" demote-non-dropping-particle="sort-only">
```
Comment (rmzelle) [inline german-council-of-economic-experts.csl]: With `default-locale="de-DE"`, the style is fixed to German. If you also would like to use the style in English, you can delete this.

File: german-council-of-economic-experts.csl (hunk 1 above)
Comment (rmzelle) [inline german-council-of-economic-experts.csl]: And did you base this on another style? If so, please add a "template" link to that style. See https://github.com/citation-style-language/styles/wiki/Style-Requirements#8---template-link

File: german-council-of-economic-experts.csl (hunk 2 above)
Comment (Irazall) [inline german-council-of-economic-experts.csl]: Thanks for your remark. I deleted it and will make a new commit

File: german-council-of-economic-experts.csl (hunk 1 above)
Comment (Irazall) [inline german-council-of-economic-experts.csl]: I added both and will make a new commit

File: german-council-of-economic-experts.csl (hunk 2 above)
Comment (rmzelle) [inline german-council-of-economic-experts.csl]: @Irazall, it doesn't look like you made this change yet.

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**